docs/guides/libraries-usage.md

```diff
@@ -0,0 +1,195 @@
+# Libraries usage
+
+You can use libraries across the app using two strategies:
+
+1. **Eager Load**: Loads the lib when the app is bootstrapped.
+2. **Lazy Loading**: Loads only when the app required it.
+
+### Eager Loading
+
+A good example was the usage of the `ui` library created previously. The lib was called directly on the `app.module.ts`
+
+### Lazy Loading
+
+You can check the feature-shell usage:
+
+## Feature Shell Libraries
+
+To illustrate how to use lazy load strategy for libraries, create a library of Angular components.
+
+**Run `ng g lib feature-shell --router --lazy`, and select Angular as the library framework.**
+
+_**Explanation**_
+
+- _**--routing**: Add router configuration. See lazy for more information._
+
+- _**--lazy**: Add RouterModule.forChild when set to true, and a simple array of routes when set to false._
+
+```treeview
+myorg/
+├── apps/
+│   ├── todos/
+│   ├── todos-e2e/
+│   └── api/
+├── libs/
+│   ├── data/
+│   ├── ui/
+│   └── feature-shell/
```

Comment:
I think it might be better to use something specific to the todos app. Maybe: feature-todo-list and have a TodoListComponent in there?

Reply:
Done.